### ISSUE TITLE
Añadir a la biblioteca

### ISSUE THREAD

**Issue by r3tr0k3r** [Palantiraddon/Repo#2]
Sería interesante tener la opción de añadir una película a la biblioteca para facilitar la integración con Kodi.

¿Hay alguna forma de ver el código sin tener q hacer el decode desde base64? No lo encuentro en github =/

**Comment by f5inet** [comment]
No. no hay forma de ver el source que no sea hacer un decode base64, y entretenerte un buen rato (de 4 a 8 horas tirando de Pycharm) con un analizador de codigo renombrando todas las variables y funciones, porque esta todo el codigo ofuscado.
